contracts/core/SenderCreator.sol

```diff
@@ -15,6 +21,9 @@ contract SenderCreator {
     function createSender(
         bytes calldata initCode
     ) external returns (address sender) {
+        if (msg.sender != entryPoint) {
+            revert("AAxx should call from EntryPoint");
```

Comment:
this is not an expected error case to happen with entrypoint, only by "side-calling".
So it is an AA9x.
Can reuse "AA92 only from entrypoint"

Reply:
Didn't want to reuse error codes, defined it s `AA97`.